### DISCUSSION TITLE
Where is the overview of the data layout?

### DISCUSSION THREAD

**Discussion by drswgw** [facebookresearch/fastMRI#241] [Q&A] [answered]
I'm a total newbie to this data set. Finding the lack of an overview of the data layout very challenging.
For instance...I would like to find one DICOM file, and the matching fully sampled K-space data that goes with THAT dicom, and download JUST those 2 sets of files.
My purpose is to understand the coil combination process, and then move forward to spiral acquisition reconstruction. In theory, I should be able to mask the fully sampled data set in k space to yield the 'equivalent' of a spiral acquired k-space data set that includes the fully sampled and the vendor reconstructed (dicom).
Just as a general comment...smaller chunks that are better documented with a README in common text for what is IN each chunk is what I was expecting. I must be missing something. It seems like the data is laid out in giant chunks with very little explanation of what is IN the chunks.
Right now I am downloading a chunk, then attempting to survey it to determine what it contains....time consuming, and quite indirect.

**Comment by mmuckley** [comment, marked as answer]
Hello @drswgw, my understanding is the information you requested for DICOM vs. fully-sampled k-space data is not available. The DICOM and raw k-space files went through independent anonymization work streams and the key for connecting them is deleted to comply with agreements for patient privacy.

We do understand the `.tar.gz` files are quite large and difficult to deal with. I would like to make them smaller, but this is taking some time as NYU controls the data and communication can be slow.

**Reply by drswgw** [reply]
That's cool, I get the patient anonymous idea, it's a good thing. 
I've been digging a little and it seems like the masked data is not what I am looking for i.e. the challenge part of the data is uninteresting to me...well that's not quite right....this type of 'banded' partial fourier is just not my 'flavor' of partial fourier sampling. I'm currently more interested in radial (like BLADE but without the redundancy) or spiral sampling. which I hope to emulate by masking from the Langone data.
While we are at it, I get the feeling there is also no 4D data (cardiac, cine, phase contrast etc.) i.e. (x,y,z,t) type data where t is time in patient space as opposed to k space....maybe that's not a totally clear distinction but I am hoping that it makes sense.
It's an AMAZINGLY RICH data set, I think I just need to put on my big boy (or big girl) pants and figure out how to work directly with the 1.2 TB data stream.....
If you could clarify...there is a comment about retiming of the kspace lines....what does that mean precisely. I would expect the kspace lines to be 'as acquired' in terms of time....but maybe that introduces artifacts?
It is attractive that the data CAN be directly FFT'd but I need to understand in just slightly more detail what that means i.e. the training set for me is not just an AI exercise, but an actual one....what we learn we hope to apply on our own research 3T Siemens Prisma MRI.

**Reply by mmuckley** [reply]
Hello @drswgw, as you can see in the [dataset paper](https://arxiv.org/pdf/1811.08839.pdf), there are only 2D scans of knees and brains (no 4D or non-Cartesian).

For working with the data, there are several examples in the repository.

Not sure what you mean about retiming of the lines - could you provide a reference?
